### ISSUE TITLE
add date sort

### ISSUE THREAD

**Issue by SGBinMB** [itsbm/RW3-Cat#1]
The first request made after putting our library on the website was to add the ability to sort by release date.  I made the change to index.php and would like to upload it to the project, but I'm new to 
Github and I'm not sure how to do it.

**Comment by itsbm** [comment]
Good idea! Maybe you could just paste the changes you made into a comment and I'll see about including it. I just made some updates to allow searching and proper display on mobile devices, but your code should still be useful.
Thanks!
